.github/workflows/qa-ec2.yml

```diff
@@ -0,0 +1,81 @@
+name: qa-ec2-e2e
+
+on:
+  # Replace with a webhook from the QA deployment in operate first
```

Comment:
You can replace this with `on: workflow_dispatch`.
That will allow us to trigger it manually or via the API etc...